gnomad_bigquery/bq_export_gnomad.py

```diff
@@ -168,23 +162,24 @@ def get_freq_struct(x: hl.expr.ArrayExpression):
         rf = v2_rf(data_type).ht()
         lcr = lcr_intervals_37.ht()
         decoy = decoy_intervals_37.ht()
-        segdup = seg_dup_intervals_37.ht() 
-    elif version == 3:
+        segdup = seg_dup_intervals_37.ht()
+    elif version == 3 and data_type == "genomes":
         mt = get_gnomad_v3_data_for_bq(
-            key_by_locus_and_alleles=True, 
+            key_by_locus_and_alleles=True,
             remove_hard_filtered_samples=False,
             adjust_sex_and_annotate_adj=True,
         )
         vep = v3_vep.ht()
         freq = v3_freq.ht()
         lcr = lcr_intervals_38.ht()
+        vqsr = get_filtering_model("vqsr_alleleSpecificTrans", split=True).ht()
```

Comment:
Entirely my bad but I gave you the wrong file here! While this has the correct VQSR run, it's before we set the threshold we wanted and thus doesn't correspond to what's in the release!
The correct file is:
gs://gnomad/annotations/hail-0.2/ht/genomes_v3/filtering/vqsr_alleleSpecificTrans.finalized.split.ht

Reply:
I think for now, it's probably just easiest to hard-code this file somewhere with a TODO to update when v3 variant QC is in `gnomad_qc`